mllib/src/main/scala/org/apache/spark/ml/util/Instrumentation.scala

```diff
@@ -85,9 +86,27 @@ private[spark] class Instrumentation[E <: Estimator[_]] private (
   }
 
   /**
+   * Log params for tuning estimators that search for the best params by wrapping another
+   * estimator and using metrics computed by an evaluator.
+   *
+   * @param estimator the inner estimator called by the tuning estimator
+   * @param estimatorParamMaps different params tried by the tuning estimator
+   * @param evaluator evaluator used to compute the metric for each estimator param value
+   */
+  def logTuningParams(
```

Comment:
moved into ValidatorParams as a class method -- let me know if you feel strongly about it being static. thanks!